.github/CONTRIBUTING.md

```diff
@@ -62,6 +64,20 @@ If you want to create a kata for some topic, start by checking the roadmap to se
 If somebody is already working on this topic, you can try to find them (using the repository issues) and coordinate with them.
 If the topic you want is not claimed, or is not on the list, go ahead and let us know you'll be working on it by creating an issue.
 
+### Testing the Katas
+
+You can use Jupyter notebook front-end of the kata you're working on to validate the kata (i.e., to check that all tasks have correct reference solutions for them, and that all tests used in the notebook actually exist in the project.
+
+To validate the kata, use the [`scripts/validate-notebooks.ps1`](../scripts/validate-notebooks.ps1) script. 
+For example, to validate BasicGates kata run the following command from the root directory of the QuantumKatas project:
+
+```bash
```

Comment:
Minor, but the quoted syntax in the line below is either PowerShell, or depends on `validate-notebooks.ps1` having the right shabang as its first line (currently, the copyright header is the first line).